CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 - `Raxx.Request` struct now has a `raw_path` field to hold the unparsed path of the URL.
 
+### Changed
```

Comment:
Hmm this has got a bit broken. The raw_path comment should be part of a release on the 22 for 0.15.1 and this needs to go above  that

Reply:
I had changes locally, This might just need rebasing against latest master